pom.xml

```diff
@@ -351,13 +351,17 @@
                 <version>${commons.logging.version}</version>
             </dependency>
 
-            <!-- springboot -->
+            <!--springboot-->
             <dependency>
                 <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-dependencies</artifactId>
+                <artifactId>spring-boot-starter-web</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-jetty</artifactId>
```

Comment:
Is it possible to move all spring dependencies to seatunnel-server module?

Reply:
hi, @dijiekstra working on this. I will close it. this is just for a test. thanks for your review